packages/camera/camera_windows/windows/camera_plugin.cpp

```diff
@@ -569,6 +604,66 @@ void CameraPlugin::StopVideoRecordingMethodHandler(
     cc->StopRecord();
   }
 }
+void CameraPlugin::StartImageStreamingMethodHandler(
+            const EncodableMap& args, std::unique_ptr<flutter::MethodResult<>> result) {
+    auto camera_id = GetInt64ValueOrNull(args, kCameraIdKey); //get camera id
+    if (!camera_id) {
+        return result->Error("argument_error",
+                             std::string(kCameraIdKey) + " missing");
+    }
+    //check if request already exists
+    auto camera = GetCameraByCameraId(*camera_id);
```

Comment:
Similar comment here (and at the equivalent spot in the stop method); please make the type explicit.